src/utils/Authorizations.ts

```diff
@@ -0,0 +1,115 @@
+import { useStore } from '@/store/store'
+import { AccountTypes, AuthorizationRoles, AuthorizedActions } from '@/enums'
+
+/**
+ * Whether the specified action is authorized for the current user.
+ * Ultimately we'll just check if the auth roles includes the specified action.
+ * @returns True or False
+ */
+export function IsAuthorized (action: AuthorizedActions): boolean {
+  switch (true) {
+    case isBusinessRegistryStaff(): return BusinessRegistryStaffRoles.includes(action)
+    case isMaximusStaff(): return MaximusStaffRoles.includes(action)
+    case isContactCentreStaff(): return ContactCentreStaffRoles.includes(action)
+    case isSbcFieldOfficeStaff(): return SbcFieldOfficeStaffRoles.includes(action)
+    default: return DefaultRoles.includes(action)
+  }
+}
+
+/**
+ * Whether the user is Business Registry Staff.
+ * Ultimately we won't need this function and we'll just check auth roles for everything.
+ */
+function isBusinessRegistryStaff (): boolean {
+  const store = useStore()
```

Comment:
So this is interesting...

When I had this at the top of this module, I got unit test errors "useStore is not a function". This is apparently because the store was not yet initialized. Moving it here, and just using the default Pinia (no need to set active Pinia), solves this problem for the unit tests and works fine otherwise.